changelog/issue-1909.md

```diff
@@ -0,0 +1,4 @@
+level: minor
+reference: issue 1909
+---
+Taskcluster UI Log pages now have the log name displayed in the header and page title.
```

Comment:
```suggestion
Taskcluster UI log viewer now displays the log name in the app bar.
```

Reply:
I committed the other suggestion via GitHub without adding to batch (oops), and I'll have to update this separately. Sorry for making too many commits on this PR.

Reply:
No worries. Just let me know when this is ready for review.

Reply:
Thanks @helfi92 it's ready for review.